stat/running/running.go

```diff
@@ -0,0 +1,69 @@
+// Copyright ©2019 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package running
+
+import "math"
+
+// Mean is a running mean accumulator.
+type Mean struct {
```

Comment:
Should we maybe name this type "MeanExpDecay" to leave the floor open for future running mean implementations?